### DISCUSSION TITLE
第二届新艾利都人民代表大会完美闭幕哩（喜

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#2422] [其他]
本次会议指出，新艾利都在空洞预防方面依旧有加强的需要，目前防卫军黑曜石军营第三小队·洛彼得勒小队已经接手零号空洞的部分管理，其他区域也会由其他分队及分营执行，同时主席指出，新艾丽都某些地方的治安状态堪忧，例如芭莱大厦空洞周边低危区域，甚至有帮派在高危区域建立点位，比如最近臭名昭著的“山狮帮”，目前，山狮帮已经成功落网，在此感谢特别隐藏人员：“***·***”，朱鸢，赛斯等人，甚至啊，前几日有一大批醉汉去治安局闹事，据悉，这群醉汉口供是说：“有人指使我们，说有免费送的菜”，其余大会相关内容可以去问问朱鸢探长或者看过直播的群众们！
谢谢大家一直以来对新艾丽都治安的信任！  
                                                                                                                                      ——11号

**Comment by HFOxin** [comment]
![image](https://github.com/user-attachments/assets/104db0a4-66da-4500-a218-778fba6d7dfd)
证据如上图，出自GitHub用户Soldier11-ObsidianBarracks，绳网名称11号于2025-01-23发布的《【法律条文】新艾利都民法典》下评论区F2楼评论区第一条回复。
现根据《新艾利都宪法》（下称宪法）《新艾利都各级人民代表大会监督法》向新艾利都人民代表大会常务委员会委员长提出质询
被质询人声明第一届新艾利都人民代表大会为秘密展开，但根据《新艾利都宪法》第五十八条：
“新艾利都人民代表大会代表名额依据内环与外环的人口数量、区域功能、民族构成等因素科学合理分配，各少数民族、希人及改造人在其中均应有适当名额代表。代表选举由新艾利都人民代表大会常务委员会主持，严格保证选举公平、公正、公开。”
质询人认为第一届人民代表大会违反了选举公开的原则，触犯了宪法，根据宪法第五条：
“新艾利都厉行依法治国，全面建设社会主义法治国家。国家维护社会主义法制的统一与尊严。一切法律、行政法规和地方性法规都不得同宪法相抵触。一切国家机关、武装力量、各政党、各社会团体、各企业事业组织都必须严格遵守宪法和法律。任何违反宪法和法律的行为，都必须予以追究。任何组织或者个人都不得有超越宪法和法律的特权。”
第一届人民代表大会的召开明显属于违宪行为，应当予以追究。
                            上述为全部质询内容，请被质询人尽快答复
